test/README.md

```diff
@@ -159,51 +159,53 @@ On how to run tests in this direcotry, see
   </tbody>
 </table>
 
-## Common module API
+## Testing Module APIs
+
+### common
 
 The common.js module is used by tests for consistency across repeated
```

Comment:
Are we still calling the module `common.js` after the refactor? How about just `common`